### ISSUE TITLE
Compiling the Solution

### ISSUE THREAD

**Issue by BandidoPabs** [CwanOcak/Gepard#7]
I can't compile the solution. Every time I try it fails, using visual studio 2019

**Comment by VictorDelac** [comment]
> 

same problem, I use visual studio code  and it shows lots of warnings .. can´t compile .. if you have any advise pls let me know

**Comment by pransnarp** [comment]
same problem has 77 errors and 0 of 92 warnings. please advise for a solution. thanks!

**Comment by akositan** [comment]
Complie it as 32bit

**Comment by Beefstetak** [comment]
> Complie it as 32bit

Do I have to change something inside the code?

**Comment by chanrio13** [comment]
any update on how to compile properly without error?

**Comment by akositan** [comment]
> > Complie it as 32bit
> 
> Do I have to change something inside the code?

No. The code is perfectly working, you just have to compile and run as 32bit.

**Comment by b3nni3** [comment]
@akositan  is this still working? I tried compiling it and ran as 32bit, but I get this error:

<img width="1505" height="89" alt="Image" src="https://github.com/user-attachments/assets/7c30ca07-9c21-4a7b-9f0b-f224c3e79015" />
